cmd/utils.go

```diff
@@ -661,10 +662,21 @@ func MoveDir(log *LoggingConfig, fromDir string, toDir string) error {
 
 // CopyDir Copies folder from source destination to target destination
 func CopyDir(log *LoggingConfig, fromDir string, toDir string) error {
+	// fail if fromDir does not exist on the file system
 	_, err := os.Stat(fromDir)
 	if err != nil {
-		log.Error.logf("Cannot find source directory %s to copy", fromDir)
-		return err
+		log.Error.logf("Source %s does not exist.", fromDir)
+		//return err
+		return errors.Errorf("Source %s does not exist.", fromDir)
+	}
+
+	// fail if toDir exists on the file system
+	// toDir should just be the name of the target directory, not an existing file or directory
+	// if toDir is an existing file or directory it causes inconsistent results between windows and non-windows
+	_, err = os.Stat(toDir)
```

Comment:
For this `os.Stat` and the one above, we should instead use `Exists()` func from utils.go. It returns a bool and err. The bool indicates if the file/dir exists or not, and the err is != nil if there was a problem checking the file/dir.

Reply:
changes incorporated